docs/windows.md

```diff
@@ -1,142 +1,94 @@
 # Building on Windows
 
-## Install Git For Windows
-
-As Git is used by the Go language to download dependencies, you need
-to install Git, for example from http://git-scm.com/.
-
-You also must make sure that the directory that contains the Git For
-Windows binary is in the Path environment variable. Note that Git For
-Windows has a 'git' binary in a 'Git\bin' directory and another one in
-a 'Git\cmd' directory. You should only put the 'Git\cmd' directory in
-the Path environment variable.
-
 ## Install Go
 
-Please install the Go language as explained on
-https://golang.org/doc/install.
-
-To properly install Go, you will need to set some environment
-variables. We recommend you to set them globally using the Control
-Panel, as explained in the documentation above, so that these
-environment variables are automatically available in all the possible
-environments that you might want to use like Git Bash, Windows's cmd,
-Cygwin's terminal, Windows' PowerShell and so on.
-
-You must make sure that the GOROOT environment variable is set and
-that the %GOROOT%/bin directory is in the Path environment variable.
-
-The GOPATH environment variable should also be set to a directory that
-you have created, and the %GOPATH/bin directory should also be in the
-Path environment variable.
-
-## Download go-ipfs and fix Git authentication
+`go-ipfs` is built on Golang and thus depends on it for all building methods.  
+https://golang.org/doc/install  
+The `GOPATH` environment variable must be set as well.  
+https://golang.org/doc/code.html#GOPATH
```

Comment:
Do you also still need to set `GOROOT`? I think the previous version of this section was a little bit more explanatory for someone who might not be familiar with building Go programs. I know when I first approached Go, I found the various environment variables it required surprising (the equivalents are typically optional customizations with many other languages).

It might make sense to keep most of the previous version of this section as-is or tweak it slightly. Keeping it short is good, but this version strikes me as a little *too* terse.

Reply:
The Go installer on Windows sets `GOROOT` for us. I don't remember the exact release that started doing this but it's certainly before our minimum required version for building. I'm leaning towards keeping mention of `GOROOT` out since it should be set already and there's a reference to the Go documentation which is updated independently to include any necessary steps to setup the environment as they change with the language.

Maybe adding a comment to reinforce/advising users read it like: "Make sure to read the documentation at https://golang.org/doc/install and https://golang.org/doc/code.html#GOPATH to ensure your environment is properly setup for building".